ansible/roles/kong-api/defaults/main.yml

```diff
@@ -5692,3 +5693,435 @@ kong_apis:
+
+ - name: createTopicOfDiscussions
+    uris: "{{ discussions_prefix }}/v2/topics"
+    upstream_url: "{{ discussions_mw_service_url }}/v2/topics"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'discussionCreate'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+ - name: createCategoryOfDiscussion
+    uris: "{{ discussions_prefix }}/v2/categories"
+    upstream_url: "{{ discussions_mw_service_url }}/v2/categories"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'discussionCreate'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+ - name: createGroupsOfDiscussion
+    uris: "{{ discussions_prefix }}/v2/groups"
+    upstream_url: "{{ discussions_mw_service_url }}/v2/groups"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'discussionCreate'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+ - name: createNewPostOfDiscussion
+    uris: "{{ discussions_prefix }}/v2/posts"
+    upstream_url: "{{ discussions_mw_service_url }}/v2/posts"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'discussionCreate'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+ - name: createNewUserOfDiscussion
+    uris: "{{ discussions_prefix }}/v2/users"
+    upstream_url: "{{ discussions_mw_service_url }}/v2/users"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'discussionCreate'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+ - name: createChatsOfDiscussion
+    uris: "{{ discussions_prefix }}/v2/users/"
```

Comment:
duplicate URLs can't be onboarded @VenkatKadiveti

Reply:
Please check the above URLs and make sure the upstream_url and uri is not having duplicate values.